### DISCUSSION TITLE
JavaScript Client Mapper Deprecated in v20+?

### DISCUSSION THREAD

**Discussion by hariubc** [keycloak/keycloak#17153] [Archive]
Is the JavaScript Client mapper deprecated in v20+? It does not appear to be there. Would there be suggestions to alternatives?

**Comment by bonnm** [comment]
https://www.keycloak.org/docs/latest/server_development/index.html#_script_providers

**Reply by hariubc** [reply]
Thank you

**Reply by bcm0** [reply]
If you manage to use it, I'd be happy to get some advice in [How to use Javascript in a practical way?](https://github.com/keycloak/keycloak/discussions/16667)
